spyder/plugins/editor/confpage.py

```diff
@@ -50,29 +50,40 @@ def setup_page(self):
         # ---- Display tab
         showtabbar_box = newcb(_("Show tab bar"), 'show_tab_bar')
         show_filename_box = newcb(
-            _("Show full file name on top of tab bar"),
+            _("Show full file path above editor"),
             'show_filename_toolbar'
         )
         showclassfuncdropdown_box = newcb(
-                _("Show selector for classes and functions"),
+                _("Show class/function selector"),
                 'show_class_func_dropdown')
+        scroll_past_end_box = newcb(_("Allow scrolling past file end"),
+                                    'scroll_past_end')
         showindentguides_box = newcb(_("Show indent guides"),
                                      'indent_guides')
         showcodefolding_box = newcb(_("Show code folding"), 'code_folding')
         linenumbers_box = newcb(_("Show line numbers"), 'line_numbers')
-        breakpoints_box = newcb(_("Show breakpoints"), 'editor_debugger_panel',
-                                section='debugger')
-        blanks_box = newcb(_("Show blank spaces"), 'blank_spaces')
+        breakpoints_box = newcb(
+            _("Show debugger breakpoints"),
+            'editor_debugger_panel',
+            section='debugger',
+        )
+        todolist_box = newcb(
+            _("Show code annotations"),
+            'todo_list',
+            tip=_(
+                "Display a marker to the left of line numbers when the "
+                "following annotations appear at the beginning of a comment: "
+                "<tt>TODO, FIXME, XXX, HINT, TIP, @todo, HACK, BUG, OPTIMIZE, "
```

Comment:
These markers work now in lowercase too, so I think we should mention that here.

Reply:
I added a mention on the line below 👍 

```diff
@@ -74,7 +74,7 @@ def setup_page(self):
                 "Display a marker to the left of line numbers when the "
                 "following annotations appear at the beginning of a comment: "
                 "<tt>TODO, FIXME, XXX, HINT, TIP, @todo, HACK, BUG, OPTIMIZE, "
-                "!!!, ???</tt>"
+                "!!!, ???</tt> (and their lowercase variants)"
             ),
         )
         currentline_box = newcb(_("Highlight current line"),
```